docs/application-connector/docs/013-details-api.md

```diff
@@ -9,3 +9,7 @@ Find the Application Connector API documentation in the included Swagger files.
 - See [this file](assets/metadataapi.yaml) for the Metadata API specification.
 
 For convenient viewing, open these files using [this](https://editor.swagger.io/) Swagger editor.
+
+The API specification for a specific version can be acquired directly from services:
```

Comment:
`You can acquire the API specification for a given version directly from the [Connector Service](https://connector-service.kyma.local/v1/api.yaml).` -> not sure whether it's technically correct